auth/auth-store/src/main/java/com/bazaarvoice/emodb/auth/permissions/TablePermissionManager.java

```diff
@@ -73,34 +69,53 @@ public TablePermissionManager(PermissionResolver permissionResolver, DataStore d
     }
 
     @Override
-    public void updateForRole(String role, PermissionUpdateRequest request) {
-        update(getRoleKey(role), request);
-    }
-
-    private void update(String key, PermissionUpdateRequest request) {
+    public void updatePermissions(String id, PermissionUpdateRequest request) {
+        checkNotNull(id, "id");
         checkNotNull(request, "request");
         validateTable();
 
```

Comment:
Checking the request's isRevokeRest() is missed here.  - is this intentional? I see that the revokeRest property was explicitly added in this PR.